<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RunnersForm</class>
 <widget class="QWidget" name="RunnersForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="KEditListWidget" name="kcfg_runnersList">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>531</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="KTitleWidget" name="runnersTitle">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>551</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Runners</string>
   </property>
   <property name="comment">
    <string>Runners for wine and wineyards.</string>
   </property>
   <property name="icon">
    <iconset theme="start">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
  </widget>
  <widget class="QCheckBox" name="kcfg_preReleaseEnabled">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>391</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Pre-release runners enabled?</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KEditListWidget</class>
   <extends>QWidget</extends>
   <header>keditlistwidget.h</header>
  </customwidget>
  <customwidget>
   <class>KTitleWidget</class>
   <extends>QWidget</extends>
   <header>ktitlewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
